carma-messenger-core/cpp_message/include/MobilityPath_Message.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+/*
+ * Copyright (C) 2020 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+#include "cpp_message.h"
+
+namespace cpp_message
+{
+    class Mobility_Path
+    {
+        private:
+        static const int MAX_POINTS_IN_MESSAGE=60; //The maximum number of points which can be included in a mobility message containing a trajectory over DSRC
+        //Location Range for x and y
+        static const long LOCATION_MIN=-638363700;
+        static const long LOCATION_MAX=638363700;
+        //Location Range for z
+        static const long LOCATION_MIN_Z=-636225200;
+        static const long LOCATION_MAX_Z=636225200;
```

Comment:
Where did this MIN_Z and MAX_Z value come from? I don't see it in the ASN.1 spec

Reply:
These values were taken from cav_msgs::LocationECEF.msg